app/controllers/tasks_controller.rb

```diff
@@ -26,6 +26,7 @@ def create
   def update
     task.assign_attributes(task_params(task.class))
     task.save!
+    task.send_emails if task.respond_to? :send_emails
```

Comment:
I don't think we'll implement a `Task#send_emails` method which always does the same thing. It wouldn't make sense for ad-hoc tasks. If `Task#send_emails` ever _does_ send emails, they'll most likely be configured dynamically at runtime.